crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/client/AuthorisationMeansAssessmentClient.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.laa.crime.meansassessment.client;
+
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+import uk.gov.justice.laa.crime.meansassessment.model.AuthorizationResponse;
+
+@Slf4j
+@Component
+public class AuthorisationMeansAssessmentClient {
```

Comment:
I can't see where this class is being used - can it be deleted?
The CreateAssessmentValidator already seems to validate the new work reason.

Reply:
Removed.